vector/src/main/java/im/vector/app/features/media/VectorAttachmentViewerAction.kt

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright 2019 New Vector Ltd
```

Comment:
Copyright `2022`*

Reply:
Thanks. I may have done a copy/paste from an older file. I will fix it.